test/react-native/features/fixtures/app/react_navigation_js/app/scenarios/ReactNavigationBreadcrumbsDisabledScenario.js

```diff
@@ -0,0 +1,68 @@
+import Scenario from './Scenario'
+import Bugsnag from '@bugsnag/react-native'
+import * as React from 'react'
+import {View, Text, Button} from 'react-native'
+import {createStackNavigator} from '@react-navigation/stack';
+
+export class ReactNavigationBreadcrumbsDisabledScenario extends Scenario {
```

Comment:
This is identical to `ReactNavigationBreadcrumbsEnabledScenario` apart from the constructor.  To save maintenance, can we change `ReactNavigationBreadcrumbsDisabledScenario` to extend `ReactNavigationBreadcrumbsEnabledScenario` and then remove everything apart from the constructor?